PLAT-129755: Sync Input with Sandstone (#483)
enactjs/agate: CHANGELOG.md

- `agate/ArcPicker` and `agate/ArcSlider` prop `disabled` to be inactive
- `agate/ArcSlider` prop `aria-valuetext` to override `aria-valuetext` for it
- `agate/Button` prop `iconOnly`, `iconPosition` and `minWidth`

> - `agate/Input` prop `iconSize`, `size`, `invalid` and `invalidMessage`

Reviewer: Please order props alphabetically

- `agate/ArcSlider` prop `aria-valuetext` to override `aria-valuetext` for it
- `agate/Button` prop `iconOnly`, `iconPosition` and `minWidth`
- `agate/DateTimePicker` prop `dayAriaLabel`, `hourAriaLabel`, `meridiemAriaLabel`, `minuteAriaLabel`, `monthAriaLabel`, and `yearAriaLabel`

> - `agate/Input` prop `iconSize`, `invalid`, `invalidMessage` and `size`

Reviewer 2: ```suggestion
- `agate/Input` prop `iconSize`, `invalid`, `invalidMessage`, and `size`
```

> - `agate/Input` prop `iconSize` to change the size of icon from `slotBefore` and `slotAfter` (default large), `invalid` to display a tooltip with a message when `invalid` prop is true, `invalidMessage` to customize the tooltip message when `invalid` prop is true, and `size` to change the size of input (default large)

Reviewer 2: ```suggestion
- `agate/Input` prop `iconSize` to change the size of icon from `slotBefore` and `slotAfter` (default large)
- `agate/Input` prop `invalid` and `invalidMessage` to display a tooltip message indicating that the text is invalid
- `agate/Input` prop `size` to change the size of input (default large)
```

I'm not sure. Please fix it when merge.

Author: I updated the changelog. Thanks for suggestion!
